tagreader.yaml

```diff
@@ -60,24 +60,98 @@ api:
     then:
     - rtttl.play: !lambda 'return song_str;'
 
+  - service: write_tag_random
+    variables:
+      continuous: bool
+    then:
+      - lambda: 'id(continous_writing_mode) = continuous;'
+      - while:
+          condition:
+            lambda: 'return id(continous_writing_mode);'
+          then:
+          - lambda: |-
+              static const char alphanum[] = "0123456789abcdef";
+              std::string uri = "https://www.home-assistant.io/tag/";
+              for (int i = 0; i < 8; i++)
+                uri += alphanum[random_uint32() % (sizeof(alphanum) - 1)];
+              uri += "-";
```

Comment:
This is optional. For HA any random string works. I guess it makes it more readable / mimics what iOS and Android write.